astropy/table/tests/test_column.py

```diff
@@ -8,6 +8,7 @@
 
 from ...tests.helper import pytest, assert_follows_unicode_guidelines
 from ... import table
+from ..table import Table
```

Comment:
Use use the existing `table` import above with `table.Table`.  Most of the time one should minimize changes and follow the existing convention in a module.